laserscan_kinect/src/laserscan_kinect_node.cpp

```diff
@@ -79,24 +79,24 @@ void LaserScanKinectNode::connectCb(const ros::SingleSubscriberPublisher& pub)
 {
     std::lock_guard<std::mutex> lock(connect_mutex_);
 
-    if (!sub_ && pub_.getNumSubscribers() > 0)
```

Comment:
was this necessary? I thought we had established that it's ok to not publish if there are no subscribers

Reply:
That is true, we probably not comment this out. I just commented it out for testing so I could see all the topics when I ran a rostopic list.